algrtm/src/app/pianoRoll/page.js

```diff
@@ -1,7 +1,28 @@
+import PianoRollDescription from "@/components/PianoRollDescription";
+import PianoRollPlay from "@/components/PianoRollPlay";
+import RoundedTextComponent from "@/components/RoundedTextComponent";
+import SolutionsSection from "@/components/SolutionsSection";
+import StoryComponent from "@/components/StoryComponent";
+import RoundedRectangle from "@/components/shared/RoundedRectangle";
 import React from "react";
 
 function PianoRoll() {
-  return <div>PianoRoll</div>;
+  return (
+    <main className="flex flex-col items-center justify-center">
+      <PianoRollDescription />
+      <PianoRollPlay />
+
+      <RoundedTextComponent
+        version="version3"
+        backgroundColor="neutralLight-neutral10"
+        textTitleColor="neutralLight-neutral100"
+        textParagraph1Color="neutralLight-neutral100"
+      />
```

Comment:
It looks like here in Figma the gap between title and paragraph is `24px`, and on some pages the gap is `32px`, so there is a need to add a small adjustment to this component with additional prop to change it

Also in Figma the paragraph color is `#B3B3B3 (Neutral-60)`